### DISCUSSION TITLE
"--15-- --7-- --3-- --1--" What does it mean?

### DISCUSSION THREAD

**Discussion by WesWesthaver** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#789] [General]
Today I was doing some maintenance on my hot tub and was standing near the spa pump and as I turned the heater/pump/bubbles on or off I saw this scroll sideways across the display: 

--15-- --7-- --3-- --1--

Everything is working fine but I had never noticed this before. What does it mean?

**Comment by metaljay** [comment]
Upon boot it usually scrolls the IP address of the board on the display.

hard reboot it (the pump) and see what it says

**Comment by visualapproach** [comment]
You have notification on. It's a countdown. See the manual

**Comment by WesWesthaver** [comment]
Page 15 of the 2023-11-20 13:38 manual says:

"Notification will make the display beep [notification time] before bubbles,
water pump or heat is changing due to scheduled action. It will make a
second beep at half that time. And half that time again, until 2 seconds
remain. The display will also show seconds to go. This feature only works
with 6-wire displays."

If I turn the heater/pump/bubbles on or off from the UI on my phone is that considered a "scheduled action"? Is every action treated like an entry in the command queue?

It appears that the requested action occurs before the countdown completes. What purpose does the countdown serve?

**Reply by visualapproach** [reply]
Someone thought it would be useful to have a heads up before their scheduled commands was running. Maybe he wanted to secure his beer from diluting before the bubbles started or something.
All commands are placed in the command queue (scheduled) but immidiate actions like button presses would (should) not trigger a notification. The text may be lagging a bit if it can't scroll faster than the count down, but it shouldn't be completely out of sync.
